libs/sdk-ui-filters/src/AttributeFilter/utils/AttributeFilterUtils.ts

```diff
@@ -137,6 +137,7 @@ export const getElementTotalCount = async (
         .forDisplayForm(objRef)
         .withOptions({
             ...(searchString ? { filter: searchString } : {}),
+            excludePrimaryLabel: true,
```

Comment:
Is this correct for bear? (or shoud we use some backend capability for this?)

Reply:
Bear does not implement this option. It is applied only on Tiger, Bear ignores it. The fact is explained in the TSDoc. Is it OK like this? I'm not sure how to handle it differently.